components/admin_console/group_settings/group_details/group_details.jsx

```diff
@@ -115,14 +132,59 @@ export default class GroupDetails extends React.PureComponent {
         this.setState({loadingTeamsAndChannels: false});
     }
 
-    onToggle = async (allowReference) => {
-        this.setState({allowReference});
-        this.props.actions.patchGroup(this.props.groupID, {allow_reference: allowReference});
+    onMentionToggle = async (allowReference) => {
+        const {group} = this.props;
+        const isOriginalAllowReference = group.allow_reference;
```

Comment:
I would prefer if this was just named `originalAllowReference`.

Reply:
sure